reporter/cloudwatch.go

```diff
@@ -162,14 +175,14 @@ func metricsData(cfg *config.Config) []*cloudwatch.MetricDatum {
 			if t.Count() == 0 {
 				return
 			}
-			dataz := map[string]float64{
-				fmt.Sprintf("%s.count", name):          float64(t.Count()),
-				fmt.Sprintf("%s.one-minute", name):     t.Rate1(),
-				fmt.Sprintf("%s.five-minute", name):    t.Rate5(),
-				fmt.Sprintf("%s.fifteen-minute", name): t.Rate15(),
-				fmt.Sprintf("%s.mean", name):           t.RateMean(),
```

Comment:
Might want to check with @zsiddique if he knows of any usage of these rate metrics.

Reply:
The only case of us using metrics with `-minute` in Graphite is for wpa-driver, on what seem to be meter metrics. (And it's not on Lambda anyway, so should be fine)

But the existing percentile naming pattern (metric-perc0.990) is referenced on a few different dashboards, so I'm going to undo that part of this change.

Reply:
And actually, maybe the rates would be useful on other things in the future - I'll add them back, although `.mean` is going to be `.rate-mean` now.

Looking at usage of `.mean` in our dashboards, every one I've looked at on timers appears to be assuming it's the mean timer value, not the mean rate (which means some of them are currently wrong until this gets merged)